### PR TITLE
netvfy gui: enable disconnect

### PR DESCRIPTION
**Pull request by si74**
While testing this, some issues that are noted: 
1. if I connect and then click disconnect before connection is completed, the whole damn thing hangs
2. if i click connect, I can still disconnect a different network (the whole list of options are there)
3. if I click connect, wait for it to connect, and then disconnect, there is no log line printed indicating success
4. if i click connect, wait for it to connect, then disconnect, and reconnect on another network, there is an error

### CODE REVIEW

File: src/gui/main.cpp [hunk 1]
```diff
@@ -165,12 +167,14 @@ void MyFrame::onClickDeleteNetwork(wxCommandEvent &event)
 	ndb_networks(this->onListNetworks);
 }
 
-/*
+
 void MyFrame::onClickDisconnect(wxCommandEvent &event)
 {
 	agent_thread_fini();
```
Comment (nicboul) [inline src/gui/main.cpp]: I'm not sure that this is working properly :|

**Comment by si74** [comment]
Remaining steps: 

- [ ] Fix weirdly large static_text_2 box at the top
- [x] should the network names only display the one that has been connected to (otherwise user may click a different one to disconnect)

**Comment by si74** [comment]
Now - logged out is displayed:
Screen Shot 2019-10-07 at 11 20 03 PM

After a user is logged out, the IP address is removed from the text box: 
Screen Shot 2019-10-07 at 11 20 08 PM

**Review by nicboul — APPROVED** [review]
LGTM

**Pull request merged**